public/components/wz-search-bar/components/wz-search-badges/context-menu.tsx

```diff
@@ -205,7 +205,7 @@ function EditFilterOperator(operator, setOperator) {
 
 function EditFilterConjuntion(conjuntion: string, setConjuntion): React.ReactNode {
   return <EuiFormRow label="Conjuntion">
-    <EuiButtonGroup options={[
+    <EuiButtonGroup legend="Conjunction" options={[
       { id: `conjuntion-AND`, label: "AND" },
       { id: `conjuntion-OR`, label: "OR" },
```

Comment:
Could we pass these options to a variable?

Reply:
sounds good, let's improve the code readability